ansible/files/runtimes.json

```diff
@@ -196,6 +196,20 @@
         "go": [
             {
                 "kind": "go:1.11",
```

Comment:
should we even keep this?

Reply:
I think we should definitely remove go:1.11 before a 1.0.0 release.

I was thinking we would do the remove in a separate PR in case downstreams wanted more control.